src/applications/gi/containers/Modals.jsx

```diff
@@ -329,7 +329,7 @@ export class Modals extends React.Component {
               target="_blank">this page.</a>
           </p>
           <p>
-            Only Veterans entitled to the maximum benefit rate or their designated transferees can receive this funding. Active duty Servicemembers and their spouses are not eligible for this program (child transferees of active duty Servicemembers may be eligible if the servicemember is qualified at the 100% rate). This information will be updated quarterly.
+            Veterans, Fry Scholarship, and Purple Heart recipients are entitled to the maximum benefit rate or their designated transferees can receive this funding. Active duty Servicemembers and their spouses are not eligible for this program (child transferees of active duty Servicemembers may be eligible if the servicemember is qualified at the 100% rate). This information will be updated quarterly.
```

Comment:
This is the requested change from stakeholders, @peggygannon would it be of interest to reword this as "Veterans and Fry Scholarship and Purple Heart recipients" if that is clearer?

Reply:
That works just fine